ddl/ddl_tiflash_test.go

```diff
@@ -60,6 +60,8 @@ const (
 func (s *tiflashDDLTestSuite) SetUpSuite(c *C) {
 	var err error
 
+	ddl.PollTiFlashInterval.Store(1000 * time.Millisecond)
```

Comment:
`PollTiFlashInterval` is now only set here and `TearDownSuite`, could you remove the atomic way?